packages/paymaster/index.ts

```diff
@@ -19,7 +19,6 @@ export interface Paymaster {
    * @param forGasEstimation Default to `false`.
    */
   requestPaymasterAndData(
-    runner: ContractRunner,
```

Comment:
[Minor] The `ContractRunner` type can be removed from the import

Reply:
Fixed in [22279d3](https://github.com/pilagod/waallet-extension/pull/121/commits/22279d37f2eda22d48f1f033afc9de8a6d613bbe)